### ISSUE TITLE
Does not compile for Due

### ISSUE THREAD

**Issue by jbdatko** [cryptotronix/cryptoauth-arduino#4]
The Atmel code assumes an AVR so it needs a hardware adaption layer to work on an ARM platform.

**Comment by q2dg** [comment]
...nor for Zero, so.

**Comment by jbdatko** [comment]
Yeah, it's any ARM based arduino. I changed the subject to indicated that. The issue is the [i2c](https://github.com/cryptotronix/cryptoauth-arduino/blob/master/src/atecc108-atmel/ecc108_i2c.c) portion, which is assuming AVR at the moment. 

The real problem is the wakeup, which requires SDA pulled low while SCL is high, which is a non-standard i2c thing. It can be done by just writing 0s. But I don't have that abstraction done in that file.
